### PR TITLE
Add 'Play to Earn' Banner.

### PR DESCRIPTION
**Pull request by Unengine**
Do not merge before the web page is opened/published.

### Description
SSIA

### How to test
1. Go to main menu.
2. Check if clicking event page proceeds into a url in this format.
`https://onboarding.nine-chronicles.com/nc-address?prefill="Current user's agent address"`

### Required Reviewers
@planetarium/9c-dev and @Namyujeong 

┆Issue is synchronized with this [Asana task](https://app.asana.com/0/1141562434100787/1200503121969477) by [Unito](https://www.unito.io)

### CODE REVIEW

**Comment by Namyujeong** [comment]
Good!
Btw, do we need this popup?
In my opinion, it would be ok to open the browser right away when user click on the event banner.

image

**Comment by Unengine** [comment]
> Good!
> Btw, do we need this popup?
> In my opinion, it would be ok to open the browser right away when user click on the event banner.
> 
> image

Do you think it's unnecessary?
I mostly experienced games that require confirms to proceed to external web page because it unfocuses game client itself.

**Comment by Namyujeong** [comment]
I agree with you.
Then plz just work on changing the pop-up title and body label.

File: nekoyume/Assets/_Scripts/UI/Module/PageView.cs [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+                _animationCoroutine = StartCoroutine(CoSmoothMovePage(content.localPosition, targetPosition));
+            }
+            else
+            {
+                if (_animationCoroutine != null)
+                {
+                    StopCoroutine(_animationCoroutine);
+                }
+                _animationCoroutine = StartCoroutine(CoSmoothMovePage(content.localPosition, _panelPosition));
+            }
```
Comment (hskim881028) [inline nekoyume/Assets/_Scripts/UI/Module/PageView.cs:97]: How about removing the else and calling it only once?
```
var endPos = _panelPosition;


if (_animationCoroutine != null)
 {
     StopCoroutine(_animationCoroutine);
}
 _animationCoroutine = StartCoroutine(CoSmoothMovePage(content.localPosition, endPos));
```

**Review by hskim881028 — APPROVED** [review]

**Review by Namyujeong — APPROVED** [review]

**Pull request merged**